### DISCUSSION TITLE
Can not connect db after upgrading to 3.0.1 from 3.0.0-beta.19.3

### DISCUSSION THREAD

**Discussion by renyuanz** [strapi/strapi#6501] [Legacy Topics]
**Describe the bug**
I am getting a db connection error while doing `yarn develop` (it's just `strapi develop`) after upgrading to `3.0.1`. And downgrade to `3.0.0-beta.19.3` makes it back to normal.

I'm thinking about database configuration got some breaking changes since `3.0.0-beta.20.3`, such as using relative path to resolve deps (I did an upgrade with this version but but some error too)

one more useful information might be my project's directory structure, it's a monorepo containing frontend(create-react-app) and backend (strapi), following this structure with yarn workspace:

```
root/
├── apps/
│   ├── frontend
│   │   └── package.json
│   └── backend
│       └── package.json
├── node_modules (# all packages are installed here)
└── package.json
```

and I got an error described in `Code snippets` section.

Could you please help or maybe do you have a related migration doc?

**Steps to reproduce the behavior**

**Expected behavior**
should start server without error.

**Screenshots**

**Code snippets**
```
$ strapi develop
[2020-06-02T02:38:41.313Z] debug ⛔️ Server wasn't able to start properly.
[2020-06-02T02:38:41.315Z] error TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at Object.load (/Users/xxx/Documents/GitHub/my_project/node_modules/strapi-database/lib/connector-registry.js:17:39)
    at DatabaseManager.initialize (/Users/xxx/Documents/GitHub/my_project/node_modules/strapi-database/lib/database-manager.js:32:21)
    at Strapi.load (/Users/xxx/Documents/GitHub/my_project/node_modules/strapi/lib/Strapi.js:274:19)
    at async Strapi.start (/Users/xxx/Documents/GitHub/my_project/node_modules/strapi/lib/Strapi.js:137:9)
error Command failed with exit code 1.
```

**System**

- Node.js version: v12.16.1
- NPM version: 6.13.4
- Strapi version: 3.0.1
- Database: mysql
- Operating system: mac

**Additional context**

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by vicnala** [comment]
Same issue here after upgrading to 3.0.1

**Comment by mehmetaydogduu** [comment]
Same issue while upgrading to 3.0.4 from 3.0.0.beta.20.3

**Reply by Pablo-Aldana** [reply]
I can confirm, just upgrated from 3.0.0.beta.20.3 to 3.0.4 and facing same problem

**Comment by Pablo-Aldana** [comment]
Ok it was easy than expected, the problem is the config folder has changed completely. As my project was small I just created a new installation and replace the config folder from the new installation to my project and fix the issue if you have specific configuration and need to merge it, follow the migration guide, don't do like me and Keep reading after "upgrading your dependencies" :P 

[https://strapi.io/documentation/v3.x/migration-guide/migration-guide-beta.20-to-3.0.0.html](https://strapi.io/documentation/v3.x/migration-guide/migration-guide-beta.20-to-3.0.0.html )

I hope this help you all

**Reply by derrickmehaffy** [reply]
You are correct, if you are upgrading you should be checking for a migration guide

**Reply by TechQuery** [reply]
This kind of **Break Changing** from **Beta version** to **Release version** is _unacceptable_ for old projects...

**Reply by derrickmehaffy** [reply]
I'm sorry you feel that way @TechQuery but it was necessary to move forward with the project
